roles/openshift_logging_eventrouter/defaults/main.yaml

```diff
@@ -2,7 +2,6 @@
 openshift_logging_eventrouter_replicas: 1
 openshift_logging_eventrouter_sink: stdout
 openshift_logging_eventrouter_nodeselector: ""
-openshift_logging_eventrouter_cpu_limit: null
```

Comment:
per the BZ, it looks like we should be able to set the cpu limit.  I think instead of removing it we should update the template to be similar to the other components [1].

[1] https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging_elasticsearch/templates/2.x/es.j2#L61

Reply:
@ewolinetz I'm in a little quandary here as for this component we chose a different deployment mechanism.  Unlike the others, we are producing and processing an Openshift template from a jinja template.  There is no way i know of to conditionally set the CPU_LIMIT the way we are setting the other vars.  Do we really want to conditionally include a Openshift template parameter and then process it?  Seems wonky

Reply:
We're configuring the nodeselector for the deployment config this way though. I was thinking we could just apply that same logic there. How often do we expect customers to create a new Eventrouter with the existing template though? Does our documentation advise users on how they would adjust their node selector if required? If so, we could expand that to also include the CPU_LIMIT.

Reply:
Updated to conditionally add CPU_LIMIT if var is defined